### PR TITLE
Beautify UI of Account + add Import button

### PR DESCRIPTION
**Pull request by Flechman**

### CODE REVIEW

File: app/src/main/java/com/github/onedirection/interoperability/gcalendar/ImportFragment.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+                             Bundle savedInstanceState) {
+        // Inflate the layout for this fragment
+        View view = inflater.inflate(R.layout.gcalendar_fragment_import, container, false);
+
+        view.findViewById(R.id.buttonGCalendarImport).setOnClickListener(this::doSomething);
+
+        return view;
+    }
+
+    private void doSomething(View ignored) {
```
Comment (hugolan) [inline app/src/main/java/com/github/onedirection/interoperability/gcalendar/ImportFragment.java:40]: why do you have this if it does nothing?

**Review by hugolan — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/github/onedirection/interoperability/gcalendar/ImportFragment.java (hunk 1 above)
Comment (Flechman) [reply app/src/main/java/com/github/onedirection/interoperability/gcalendar/ImportFragment.java:40]: Because it's @Ef55 ´s task

Comment (hugolan) [reply app/src/main/java/com/github/onedirection/interoperability/gcalendar/ImportFragment.java:40]: ok!

**Review by hugolan — APPROVED** [review]
nice code

**Review by Ef55 — APPROVED** [review]

**Pull request merged**